core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-box/template-builder-box.component.ts

```diff
@@ -31,9 +48,22 @@ export class TemplateBuilderBoxComponent {
     deleteContainer: EventEmitter<void> = new EventEmitter<void>();
     @Output()
     deleteColumn: EventEmitter<void> = new EventEmitter<void>();
+    @Output()
+    deleteColumnRejected: EventEmitter<void> = new EventEmitter<void>();
 
     @Input() items: DotTemplateBuilderContainer[];
 
-    protected readonly templateBuilderSizes = TemplateBuilderBoxSize;
-    @Input() size: TemplateBuilderBoxSize = TemplateBuilderBoxSize.large;
+    @Input() size = 1;
+
+    boxSize = TemplateBuilderBoxSize.small;
```

Comment:
Why size and boxSize? seems redundant to me.

Reply:
agree, changing to `boxVariant`